packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -37,6 +40,14 @@ interface IButtonClassNames {
 
 export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState> implements IButton {
 
+  private get _isSplitButton(): boolean {
+    return (!!this.props.menuProps && !!this.props.onClick);
+  }
+
+  private get _isExpanded(): boolean {
```

Comment:
This PR is still missing re-using this _isExpanded function here:

    } else if (this.props.menuProps) {
      assign(
        buttonProps,
        {
          'onClick': this._onToggleMenu,
          'aria-expanded': this.state.menuProps ? true : false,
          'aria-owns': this.state.menuProps ? this._labelId + '-menu' : null,
          'aria-haspopup': true
        }